templates/puppet_webhook.service.epp

```diff
@@ -8,7 +8,11 @@ After=syslog.target network.target
 
 [Service]
 Type=simple
+<% if $facts['osfamily'] == 'Debian' {-%>
```

Comment:
During tests structured facts are used. Why use string facts here ?

Reply:
Wouldn't it be better to use a variable for the file path? So use `$env_path` as done in managing the file.